src/components/InputTime/Dropdown/Dropdown.tsx

```diff
@@ -24,12 +24,14 @@ interface DropdownProps {
   disabled?: boolean;
   formatTime?: (date: Date) => string;
   max?: string;
+  menuProps?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
   min?: string;
   onSelectMenuItem: Function;
   showDropdown?: 'click' | 'focus';
   step?: number;
   stepFrom?: string;
   toggleAriaLabel?: string;
+  toggleProps?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
   value?: string;
   [key: string]: any; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
looks like this isn't the first time we've overridden this rule. should we just allow explicit any's project wide?

Reply:
I think it's still useful.
If the consumed components had been TS already, I would have used those instead of `Record<string, any>`.
As it is, I'd rather wait for the conversion than re-type them here.